src/bin/tikv-server.rs

```diff
@@ -198,6 +198,14 @@ fn get_rocksdb_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions
     };
     opts.set_min_write_buffer_number_to_merge(min_write_buffer_number_to_merge as i32);
 
+    let max_background_threads = get_integer_value("",
```

Comment:
The most important is to call `SetBackgroundThreads(total_threads, Env::LOW)` to make the environment use more threads for compactions and flushes.
By default, rocksdb use only one thread for background jobs. Change the `max_background_compactions` or `max_background_flushes` is useless unless you increase the total number of threads of the environment's thread pool.
Reference: [https://github.com/facebook/rocksdb/wiki/RocksDB-Tuning-Guide#parallelism-options](parallelism-options)